app/views/products/_cvu_plus_product.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="row">
+  <div class = 'col-md-8'>
+    <%= render 'product_status_table', :product => @product %>
+  </div>
+
+  <div class = 'col-md-4'>
+    <% unless @product.product_tests.empty? %>
+      <div class = 'panel panel-default'>
+        <div class = 'panel-heading'>
+          <h1 class='panel-title'>Download Full Test Deck</h1>
+        </div>
+        <div class = 'panel-body' id = 'display_bulk_download'>
+          <%= render 'bulk_download', :product => @product %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</div>
+
+<div class="product-test-tabs">
+
+<div class="product-test-tabs">
+  <ul>
+    <% each_tab(@product) do |test_type, title, _, html_id| %>
+      <li><a href = '<%= "##{html_id}" %>'><%= title %></a></li>
+    <% end %>
+  </ul>
+  <% each_tab(@product) do |test_type, title, description, html_id| %>
```

Comment:
Isn't there only one tab in this scenario? Or are we anticipating adding additional test types as CVU+ matures?